iidrees/Events-Manager
# #157342004 implement signout

#### What does this PR do?
Adds a signout option for a user who wants to signout of the application

#### Description of Task to be completed?
N/A
#### How should this be manually tested?
Clone the repo
From the project root directory 
run `npm start` 
signup or signin into the application as a user
On the navigation bar a signup link is shown.

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
 #157342004 


#### Screenshots (if appropriate)

<img width="1113" alt="screen shot 2018-05-16 at 5 58 11 pm" src="https://user-images.githubusercontent.com/24682326/40132433-ae6e6876-5934-11e8-89c4-9edc3ae366f4.png">
#### Questions:
N/A
